x-team/auto-ui
# Add Shift + Enter key combination support for InlineEditor

**Release Type:** *Non-Breaking Feature* 


Fixes https://x-team-internal.atlassian.net/browse/XP-2492

## Description

Added the possiblitiy to save the `InlineEditor` content only when the `shift + enter` combination will be performed.


## Checklist

**Before submitting a pull request,** please make sure the following is done:


- [x] component's documentation (`.stories.js` file) is changed or added accordingly to reflect any new or up
dated use cases or variants usage.

## Related PRs



branch | PR
------ | ------
XP-2492-move-cursor-to-next-line | [Auto](https://github.com/x-team/auto/pull/1699)

## Steps to Test or Reproduce

1. Edit notes and perform hit the enter key, it should affect with adding new line instead of saving the note. 
2. To perform the save use the `Shift+Enter` key combination

## Impacted Areas in Application

`InlineEditor`, `Note`
